Stronger/Info/SummaryView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Summary.swift
+//  Stronger
+//
+//  Created by Theodore Kanell on 2/1/24.
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziAccount
+import SpeziMockWebService
+import SwiftUI
+
+struct SummaryView: View {
```

Comment:
Generally looks like this view is using hard coded data and values. Please be sure that you either address this in this PR or at least by the end of the class. It is generally not advised to merge code with hard coded values.